types/react-virtualized/dist/es/Table.d.ts

```diff
@@ -229,7 +229,7 @@ export type TableProps = GridCoreProps & {
     /** Optional id */
     id?: string;
     /** Optional renderer to be used in place of table body rows when rowCount is 0 */
-    noRowsRenderer?: () => void;
+    noRowsRenderer?: () => JSX.Element;
```

Comment:
If the default is `() => null`, wouldn't this be more correct when used with `--strictNullChecks`?

```suggestion
    noRowsRenderer?: () => JSX.Element | null;
```

Reply:
Good feedback! I've committed your suggested change. TY